compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeResolver.java

```diff
@@ -0,0 +1,2155 @@
+/*
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
Shouldn't this be `WSO2 LLC.`?
```suggestion
 *  WSO2 LLC. licenses this file to you under the Apache License,
```